align/pnr/toplevel.py

```diff
@@ -230,6 +362,7 @@ def route( *, DB, idx, opath, adr_mode, PDN_mode, router_mode, selection=None):
     results_name_map = {}
 
     router_engines = { 'top_down': route_top_down,
+                       'psuedo_bottom_up': route_psuedo_bottom_up,
```

Comment:
psuedo or pseudo?

Reply:
Thanks. When I  typed it on the command line and it was always wrong.